tools/ports/vorbis.py

```diff
@@ -33,11 +33,15 @@ def create(final):
                      ['-s', 'USE_OGG=1'], ['psytune', 'barkmel', 'tone', 'misc'])
     ports.install_header_dir(os.path.join(source_path, 'include', 'vorbis'))
 
-  return [shared.Cache.get_lib('libvorbis.a', create)]
+  return [
+    shared.Cache.get_lib('libvorbis.a', create),
+    shared.Cache.get_lib('libvorbisfile.a', create),
+  ]
```

Comment:
These both call the same method `create`, so they are identical in content. But isn't libvorbisfile a library with additional code?

Reply:
Doh! So a change from `-lvorbisfile` to `-lvorbis` should be enough for my use case? :facepalm:

Sorry, I made this change without really understanding the code and things started working after this, didn't occur to me. :)